wiki/Article_Styling_Criteria/en.md

```diff
@@ -82,174 +103,303 @@ tags:
+
+### Headings
+
+Limit to a heading level of 5 and never use headings to style or format text. Prefer the [ATX (hash) style](https://github.github.com/gfm/#atx-headings) over the [Setext (underline) style](https://github.github.com/gfm/#setext-headings).
+
+Optionally, add an extra line before and after the heading to help make the heading stand out while editing.
+
+#### Titles
+
+All articles must have one level 1 heading, being the article's title. Article titles must use title case and never contain formatting, links, nor images. The title of an article is the name of the folder that the article is located in.
+
+The level 1 heading must be placed at the start of the article unless you have reference links or metadata placed there. If this is the case, place the level 1 heading underneath it.
+
+For English article titles, if you need to reword the title, you must rename the folder to match the article title. There are rare exceptions to this however.
+
+#### Sections
+
+All section headings are to be using levels 2 to 5. Section headings are to use sentence case and should never contain styles nor links. In addition to this, section headings must never skip a heading level (e.g. do not go from a level 2 heading to a level 4 heading). The first section heading must be a level 2 heading. Unlike titles, section headings can have small image icons.
+
+In the osu! wiki, heading levels 4 and 5 are hidden in the table of contents.
+
 ### Lists
 
-There are two kinds of lists: bulleted and numbered. The spacing for both types of bullets is very important! Please limit to 4 indentations/sub-bullets.
+There are two kinds of lists: bulleted and numbered. The spacing for both types of bullets is very important! Limit to 4 indentations/sub-bullets.
 
 #### Bulleted
 
-Use bulleted lists when the order of the list does not matter. It is preferred to use the hyphen (`-`) or plus symbol (`+`) instead of the asterisk (`*`) because the single asterisk is already used for italics.
+Use bulleted lists when the order of the list does not matter. It is preferred to use the hyphen (`-`) or the plus symbol (`+`) instead of the asterisk (`*`) because the single asterisk is already used for italics.
 
 #### Numbered
 
-Use numbered lists when the order of the list does matter. Incrementing the numbers is optional.
+Use numbered lists when the order of the list does matter. Do not increment the numbers. For example:
+
+```
+Game modes of osu! (in order of implementation):
+
+1. osu!standard
+1. osu!taiko
+1. osu!catch
+1. osu!mania
+```
 
 #### Mixed
 
-You can also combine both bulleted and numbered lists.
+You could combine both bulleted and numbered lists, but this should be done sparingly.
 
 ### Code
 
 The markup for code is a grave mark (`` ` ``). To put a grave mark in code, surround it in double grave marks, padded with 1 space.
```

Comment:
L348: Elaborate further. Double back-ticks for single lide codeblocks, triple back-ticks for large code blocks.

Reply:
Just to note back-ticks are another term for tildes "`"